packages/alfa-math/src/vector.ts

```diff
@@ -1,51 +1,81 @@
-import * as math from "mathjs";
+import { Real } from "./real";
 
-export type Vector<N extends number = number> = [number, ...Array<number>] & {
-  readonly length: N;
-};
+const { sqrt } = Math;
+
+/**
+ * @see https://en.wikipedia.org/wiki/Vector_(mathematics_and_physics)
+ */
+export type Vector = Array<number>;
```

Comment:
Same here: we're losing a lot of type safety.

Reply:
See https://github.com/Siteimprove/alfa/pull/268#discussion_r441754282.